apps/ecommerce/frontend/src/App.css

```diff
@@ -0,0 +1,3 @@
+body {
+  overflow: hidden;
+}
```

Comment:
fixes flickering scrollbar when field editor height changes

Reply:
flicker bug before:
https://user-images.githubusercontent.com/492573/236909197-940fe724-8614-44f1-a3f3-3f030c751971.mov

smooth and silk after:
https://user-images.githubusercontent.com/492573/236909341-4476daa2-7820-4882-bc47-e1d29c1b706e.mov